socless/integrations.py

```diff
@@ -219,22 +219,36 @@ def __init__(self, event, lambda_context, integration_handler, include_event=Fal
             self.event = event
 
         self.testing = bool(self.event.get("_testing"))
+        self.execution_id = self.event.get("execution_id", "")
+
         try:
             self.state_config = self.event["State_Config"]
-        except:
-            raise KeyError("No State_Config was passed to the integration")
+        except KeyError:
+            # not triggered from socless playbook (direct invoke via CLI, Test console, etc.)
+            if "execution_id" not in self.event and "artifacts" not in self.event:
+                print(
+                    "No State_Config was passed to the integration, likely due to invocation \
+from outside of a SOCless playbook. Running this lambda in test mode."
+                )
+                self.testing = True
+                self.state_config = {"Name": "direct_invoke", "Parameters": self.event}
+                self.event = {
+                    "_testing": True,
+                    "State_Config": self.state_config,  # maybe this will fix it?
+                }
+            else:
+                raise KeyError("No State_Config was passed to the integration")
```

Comment:
Let's start defining custom SOCless errors for these in an `exceptions.py` file. Proposing `SoclessBootstrapError` or `SoclessIntegrationError` for this one. Or something more descriptive that lets us know what part of SOCless generated this error